apps/convex/convex/vibes.ts

```
@@ -253,7 +253,7 @@ export const getUserReactedVibes = query({
       .collect();
 
     // Get unique vibe IDs that the user has reacted to
-    const reactedVibeIds = [...new Set(userReactions.map((r) => r.vibeId))];
+    const reactedVibeIds = Array.from(new Set(userReactions.map((r) => r.vibeId)));
```

ESLint

[error] Replace `new·Set(userReactions.map((r)·=>·r.vibeId))` with `⏎······new·Set(userReactions.map((r)·=>·r.vibeId))⏎····`

(prettier/prettier)